### PR TITLE
chore: update readme for Q2 2018

### LINKED ISSUE #10805: docs(readme): update stuff for Q2 2018
#### Bug, feature request, or proposal:

Proposal

#### What is the expected behavior?
Up to date content in readme file.

#### What is the current behavior?
The current [README.md](https://github.com/angular/material2/blob/master/README.md) on `master` has the 'High level stuff planned for Q1 2018 (January - March)' and should be updated to Quarter 2 of 2018.

#### What are the steps to reproduce?
Just visit the https://github.com/angular/material2 and read the *Project status*.

#### What is the use-case or motivation for changing an existing behavior?
Since version 6 of Material2 is just around the corner with a lot of great stuff, it's good to keep people up to date with the plan(s) of adding or updating features. Related issue: #9192 (for Q1 2018).

### PR DESCRIPTION
**Pull request by jelbourn**
Fixes #10805

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,15 +26,13 @@ and which pieces are blocked) and make a comment.
...
-* Research and exploration for data visualization
-* cdkTree and matTree
-* cdk/dialog
-* Switch build to bazel
-* Overlay positioning improvements
-* Research and prototyping for future Material Design updates
-* Research and prototyping drag/drop features for the CDK
+High level stuff planned for Q2 2018 (April - June):
+* Implementing CDK virtual-scroll
+* Implementing cdk drag-and-drop
```
Comment (donroyco) [inline README.md]: cdk » CDK

File: README.md [hunk 2]
```diff
@@ -86,11 +84,11 @@ High level stuff planned for Q1 2018 (January - March):
 |------------------|-------------------------------------|--------------|----------------|
 | tree             |      In-progress ([sneak peek][31]) |           -  |  [#3175][3175] |
```
Comment (aitboudad) [inline README.md]: I think `tree` will it be released with `6.0.0` ?

File: README.md (hunk 1 above)
Comment (jelbourn) [reply README.md]: Lower-cased consistently instead

File: README.md (hunk 2 above)
Comment (jelbourn) [reply README.md]: Yep

**Pull request merged**